packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -341,15 +341,29 @@ export class FileSystem extends FileSystemBase {
     const encrypted = props.encrypted ?? (FeatureFlags.of(this).isEnabled(
       cxapi.EFS_DEFAULT_ENCRYPTION_AT_REST) ? true : undefined);
 
+    // LifecyclePolicies is an array of lists containing a single policy
+    let lifecyclePolicies = [];
+
+    const lifecyclePolicy = props.lifecyclePolicy ? {
```

Comment:
Can this be simplified to just

```ts
if (props.lifecyclePolicy) {
  lifecyclePolicies.push({ transitionToIa: props.lifecyclePolicy });
}
```

Same for `outOfInfrequentAccessPolicy`

Reply:
Done.